client/js/widgets/seat.js

```diff
@@ -42,6 +42,15 @@ class Seat extends Widget {
     this.updateLinkedWidgets(true);
   }
 
+  childrenTarget() {
+    if (widgets.get(this.get("hand"))) {
+      return widgets.get(this.get("hand")).children().filter(c=>!c.get('owner') || c.get('owner')==this.get("player"));
+    } else {
+      return {}
```

Comment:
nit: I think this should return an array - that's the type we get from children() and would get from the above branch.

Reply:
Done